contrib/src/templates/mod.rs

```diff
@@ -219,22 +248,29 @@ impl Template {
     ///
     /// ```rust
     /// use std::collections::HashMap;
+    ///
+    /// extern crate rocket;
+    /// extern crate rocket_contrib;
+    ///
     /// use rocket_contrib::Template;
     ///
-    /// // Create a `context`. Here, just an empty `HashMap`.
-    /// let mut context = HashMap::new();
+    /// fn main() {
+    ///     let rocket = rocket::ignite().attach(Template::fairing());
```

Comment:
This method will be used almost exclusively during testing. As such, we should probably use a `Client` here, and call `client.rocket()` for illustrative purposes.